roofit/roofitcore/src/RooAbsCollection.cxx

```diff
@@ -1614,13 +1570,47 @@ void RooAbsCollection::sort(Bool_t reverse) {
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Sort collection topologically: the servers of any RooAbsArg will be before
+/// that RooAbsArg in the collection. Will throw an exception if servers
```

Comment:
I have not understood this, maybe re-phrase: 
Will throw an exception if servers are missing the collection

Reply:
Ok, I'll update this